Makefile

```diff
@@ -0,0 +1,229 @@
+BASE_PACKAGES := utils db settings visualization
+
+MODELS := biocro clm45 dalec ed gday jules linkages \
+	      lpjguess maat maespa preles sipnet
+
+
+MODULES := allometry assim.batch assim.sequential benchmark \
+	       data.atmosphere data.hydrology data.land \
+	       data.mining data.remote emulator meta.analysis \
+	       photosynthesis priors rtm uncertainty
+
+.PHONY: all ${BASE_PACKAGES} ${MODELS} ${MODULES}
+
+all: .install PEcAn.all ${BASE_PACKAGES} ${MODELS} ${MODULES}
```

Comment:
If the .install is the hack that allows make to know when packages are updated, then that's probably all than would need to be removed. I don't think you have to actually uninstall the packages.